lightning-c-bindings/src/version.rs

```diff
@@ -0,0 +1,8 @@
+#[no_mangle]
```

Comment:
Please leave this file out of CI.

Reply:
Do you mean out of the git repository? Or to do something in the build.yml file to exclude it?

Reply:
Leave it out of git/in .gitignore. Otherwise its recursive to try to keep it up to date - by committing changes to it you have to change it cause the git commit has changed :p